public/app/plugins/datasource/loki/querybuilder/components/LabelBrowserModal.tsx

```diff
@@ -36,22 +45,28 @@ export const LabelBrowserModal = (props: Props) => {
     onClose();
   };
 
+  const hasLogLabels = datasource.languageProvider.getLabelKeys().length > 0;
+
```

Comment:
This is still not in a place where the `languageProvider` was definitely started, or at least finished starting. Best would be to move this to the `useEffect` callback.